treelite.sh

```diff
@@ -37,9 +37,9 @@ module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@
 # Dependencies
 module load BASE/1.0
 # Our environment
-set osname [uname sysname]
-prepend-path PATH \$::env(BASEDIR)/$PKGNAME/\$version/bin
-prepend-path ROOT_INCLUDE_PATH \$::env(BASEDIR)/$PKGNAME/\$version/include
-prepend-path ROOT_INCLUDE_PATH \$::env(BASEDIR)/$PKGNAME/\$version/runtime/native/include
-prepend-path LD_LIBRARY_PATH \$::env(BASEDIR)/$PKGNAME/\$version/lib
+set TREELITE_ROOT \$::env(BASEDIR)/$PKGNAME/\$version/bin
```

Comment:
Ciao @ktf,
I totally missed this, but actually this line is wrong. @pfecchio and @fcatalan92 realised by testing the new release.
I am opening PR to fix this.

Cheers,
MAx